tests/metrics/test_heuristics_metrics.py

```diff
@@ -67,17 +69,25 @@ def test_randomization_metric(
     tmp_path,
     request,
 ):
```

Comment:
Can we add some additional (tiny) architectures to conftest.py, and add a couple of parameterizations (extra items in the parametrization list) with these architectures? Maybe the easiest would be to create those tiny architectures manually, and not train them.